src/bowlingGame.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+class BowlingGame {
+  constructor() {
+    this.rolls = [];
+  }
+
+   roll(downPins) {
+     this.rolls.push(downPins);
+   }
+
+   score() {
+     var array = this.rolls
+     var total = array.reduce(function(a, b) {
+       return a + b;
+       }, 0);
+     return total; 
+   }
+
```

Comment:
Overall a good start to the bowling challenge, I think you were on the right lines with this but maybe just not 100% sure what direction to move forward with. This Challenge benefits from real heavy planing - what do I need to keep track of? frames, turn, score, spares, strikes... By knowing this all in advance it helps moving forward.